src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -4560,6 +4560,19 @@ private MemberAst ClassMemberRule(string className, out List<Ast> astsOnError)
             return null;
         }
 
+        private bool CanBeSimpleNameToken(ref Token token)
```

Comment:
How about naming it as `TryUseTokenAsSimpleName`?

Reply:
```suggestion
        private bool TryUseTokenAsSimpleName(Token token)
```